<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Sai Kumar Manakan</author>
 <class>Interface</class>
 <widget class="QMainWindow" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>496</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>PixShot</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources/resources.qrc">
    <normaloff>:/images/images/systray-small.png</normaloff>:/images/images/systray-small.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="tabArea" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="toolFrame">
          <property name="minimumSize">
           <size>
            <width>57</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>57</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::WinPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QToolButton" name="pb_Rectangle">
           <property name="geometry">
            <rect>
             <x>2</x>
             <y>45</y>
             <width>50</width>
             <height>30</height>
            </rect>
           </property>
           <property name="toolTip">
            <string>Draw Rectangles</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../Resources/resources.qrc">
             <normaloff>:/images/images/rectangle.png</normaloff>:/images/images/rectangle.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>36</width>
             <height>22</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QToolButton" name="pb_Arrow">
           <property name="geometry">
            <rect>
             <x>2</x>
             <y>116</y>
             <width>50</width>
             <height>30</height>
            </rect>
           </property>
           <property name="toolTip">
            <string>Draw Arrow</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../Resources/resources.qrc">
             <normaloff>:/images/images/arrow.png</normaloff>:/images/images/arrow.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>31</width>
             <height>17</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QToolButton" name="pb_Settings">
           <property name="geometry">
            <rect>
             <x>6</x>
             <y>300</y>
             <width>48</width>
             <height>29</height>
            </rect>
           </property>
           <property name="toolTip">
            <string>Change Item Color and Boder</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../Resources/resources.qrc">
             <normaloff>:/images/images/settings.png</normaloff>:/images/images/settings.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonIconOnly</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QToolButton" name="pb_Circle">
           <property name="geometry">
            <rect>
             <x>2</x>
             <y>80</y>
             <width>50</width>
             <height>30</height>
            </rect>
           </property>
           <property name="toolTip">
            <string>Draw Circles</string>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../Resources/resources.qrc">
             <normaloff>:/images/images/ellipse.png</normaloff>:/images/images/ellipse.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QToolButton" name="pb_clear">
           <property name="geometry">
            <rect>
             <x>6</x>
             <y>262</y>
             <width>41</width>
             <height>31</height>
            </rect>
           </property>
           <property name="toolTip">
            <string>Clear Items</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../Resources/resources.qrc">
             <normaloff>:/images/images/clear.png</normaloff>:/images/images/clear.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
           <property name="arrowType">
            <enum>Qt::NoArrow</enum>
           </property>
          </widget>
          <widget class="QToolButton" name="pb_text">
           <property name="geometry">
            <rect>
             <x>2</x>
             <y>150</y>
             <width>50</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="toolTip">
            <string>Write Text</string>
           </property>
           <property name="text">
            <string>T</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QToolButton" name="pb_crop">
           <property name="geometry">
            <rect>
             <x>2</x>
             <y>185</y>
             <width>50</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="toolTip">
            <string>Crop</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../Resources/resources.qrc">
             <normaloff>:/images/images/crop.png</normaloff>:/images/images/crop.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QToolButton" name="pb_highligter">
           <property name="geometry">
            <rect>
             <x>2</x>
             <y>220</y>
             <width>49</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="toolTip">
            <string>Highlight</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../Resources/resources.qrc">
             <normaloff>:/images/images/highlighter.png</normaloff>:/images/images/highlighter.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QToolButton" name="pb_Pointer">
           <property name="geometry">
            <rect>
             <x>2</x>
             <y>11</y>
             <width>51</width>
             <height>30</height>
            </rect>
           </property>
           <property name="toolTip">
            <string> Move Items</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../Resources/resources.qrc">
             <normaloff>:/images/images/pointer.png</normaloff>:/images/images/pointer.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>36</width>
             <height>22</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicsView">
          <property name="contextMenuPolicy">
           <enum>Qt::NoContextMenu</enum>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="renderHints">
           <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>769</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="separator"/>
    <addaction name="actionCheck_for_Updates"/>
    <addaction name="actionPixShot_Homepage"/>
   </widget>
   <widget class="QMenu" name="menuCapture">
    <property name="title">
     <string>Capture</string>
    </property>
    <addaction name="actionScreenCapture"/>
    <addaction name="actionRegionCapture"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuCapture"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/images/images/open.png</normaloff>:/images/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/images/images/close.png</normaloff>:/images/images/close.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/images/images/copy.png</normaloff>:/images/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/images/images/paste.png</normaloff>:/images/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/images/images/save.png</normaloff>:/images/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionScreenCapture">
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/images/images/screenCapture.png</normaloff>:/images/images/screenCapture.png</iconset>
   </property>
   <property name="text">
    <string>Screen</string>
   </property>
  </action>
  <action name="actionRegionCapture">
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/images/images/regionCapture.png</normaloff>:/images/images/regionCapture.png</iconset>
   </property>
   <property name="text">
    <string>Region</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About PixShot</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionCheck_for_Updates">
   <property name="text">
    <string>Check for Updates</string>
   </property>
  </action>
  <action name="actionPixShot_Homepage">
   <property name="text">
    <string>PixShot Website</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionInsert">
   <property name="text">
    <string>Insert</string>
   </property>
  </action>
  <action name="actionDetails">
   <property name="text">
    <string>Details</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
